### DISCUSSION TITLE
possible broken compatibility with sqlcipher v3

### DISCUSSION THREAD

**Discussion by esguti** [pawelsalawa/sqlitestudio#4030] [Q&A]
### Details
It is not possible to open a database made with sqlcipher v3.15.2 (default Ubuntu version)

### Steps to reproduce
1. select SQLCipher as database type
2. select the .db file
3. introduce the password

### Operating system
Ubuntu 20.04.2 LTS

### SQLiteStudio version
impossible to open with **v3.3.2**, but prior v3.2.1 was possible to open it

**Comment by pawelsalawa** [comment]
Add `PRAGMA cipher_compatibility = 3;` to connection options:
![image](https://user-images.githubusercontent.com/481846/110756524-9db4e180-824a-11eb-9172-ec723093090f.png)

**Comment by esguti** [comment]
It works!! thank you
